tests/src/test/scala/com/evolutiongaming/scassandra/CassandraSpec.scala

```diff
@@ -20,9 +22,18 @@ import org.scalatest.wordspec.AnyWordSpec
 
 class CassandraSpec extends AnyWordSpec with BeforeAndAfterAll with Matchers {
 
-  private val config = CassandraConfig.Default
+  private lazy val config = 
+    CassandraConfig.Default.copy(
+      contactPoints = Nel(cassandraContainer.containerIpAddress),
```

Comment:
doesn't it make it more error-prone that there's a forward (though lazy) reference to `cassandraContainer`? maybe we could swap these two vals?